pkg/apis/sessions/session_state.go

```diff
@@ -59,78 +64,79 @@ func (s *SessionState) String() string {
 	return o + "}"
 }
 
-// EncodeSessionState returns string representation of the current session
-func (s *SessionState) EncodeSessionState(c encryption.Cipher) (string, error) {
-	var ss SessionState
-	if c == nil {
-		// Store only Email and User when cipher is unavailable
-		ss.Email = s.Email
-		ss.User = s.User
-		ss.PreferredUsername = s.PreferredUsername
-	} else {
-		ss = *s
-		for _, s := range []*string{
-			&ss.Email,
-			&ss.User,
-			&ss.PreferredUsername,
-			&ss.AccessToken,
-			&ss.IDToken,
-			&ss.RefreshToken,
-		} {
-			err := into(s, c.Encrypt)
-			if err != nil {
-				return "", err
-			}
+// EncodeSessionState returns an encrypted, lz4 compressed, MessagePack encoded session
+func (s *SessionState) EncodeSessionState(c encryption.Cipher, compress bool) ([]byte, error) {
+	packed, err := msgpack.Marshal(s)
+	if err != nil {
+		return nil, errors.Wrap(err, "error marshalling session state to msgpack")
```

Comment:
Please use the built in errors package if you are going to wrap errors, preferred style is `fmt.Errorf("<descr>: %w", err)`. Is there any reason to wrap this error? Are we unwrapping it later somewhere?